src/validation.cpp

```diff
@@ -939,6 +939,8 @@ bool GetTransaction(const uint256 &hash, CTransactionRef &txOut, const Consensus
                 return error("%s: txid mismatch", __func__);
             return true;
         }
+
+        return false;
```

Comment:
Mind to add a tiny comment to explain why fAllowSlow can be ignored?

Reply:
Done.